include/portability/cmsis_os.h

```diff
@@ -163,15 +164,19 @@ typedef struct os_mailQ_cb *osMailQId;
 /// Thread Definition structure contains startup information of a thread.
 /// \note CAN BE CHANGED: \b os_thread_def is implementation specific in every CMSIS-RTOS.
 typedef struct os_thread_def  {
-  os_pthread               pthread;    ///< start address of thread function
-  osPriority             tpriority;    ///< initial thread priority
-  uint32_t               instances;    ///< maximum number of instances of that thread function
-  uint32_t               stacksize;    ///< stack size requirements in bytes; 0 is default stack size
-  void                  *stack_mem;    ///< pointer to array of stack memory
-  struct k_thread       *cm_thread;    ///< pointer to k_thread structure
-  struct k_poll_signal *poll_signal;
-  struct k_poll_event   *poll_event;
-  int32_t            signal_results;
+	os_pthread               pthread;    /* start address of thread function */
+	osPriority             tpriority;    /* initial thread priority */
+	/* maximum number of instances of that thread function */
+	uint32_t               instances;
+	/* stack size requirements in bytes; 0 is default stack size */
+	uint32_t               stacksize;
+	void                  *stack_mem;    /* pointer to array of stack memory */
+	struct k_thread       *cm_thread;    /* pointer to k_thread structure */
+	struct k_poll_signal *poll_signal;
+	struct k_poll_event   *poll_event;
+	int32_t            signal_results;
+	/* a bitarray used to indicate whther the thread is used or not, 0: unused, 1: used */
+	void                        *mask;
```

Comment:
Is it better to rename it with "statusmask" ?

Reply:
yes, I updated it to use status_mask.